cascade/src/main/java/me/saket/cascade/CascadePopupWindow.kt

```diff
@@ -54,10 +56,10 @@ open class CascadePopupWindow @JvmOverloads constructor(
       background = themeAttrs.popupBackground
       clipToOutline = true
 
-      if (SDK_INT == 21) {
+      if (SDK_INT == LOLLIPOP || SDK_INT == LOLLIPOP_MR1) {
```

Comment:
Can you change these back to 21 and 22? Android's SDK has always used numbers for referring to minimum SDK level supported by APIs so it's easier to map them instead of using their names.

Reply:
I see, then I would also rename the event class to `Api21And21EventDelegate` for consistency, right?

Reply:
Yeah sounds good!

Reply:
Fixed.